integration-tests/configs/otel-gen-client/go.mod

```diff
@@ -0,0 +1,39 @@
+module github.com/open-telemetry/opentelemetry-collector-contrib/examples/demo/client
```

Comment:
Can we avoid the `go.mod` files altogether? Ideally we would bring in this dependency in an idiomatic way in our main go.mod. Is there a reason why it cannot be done?

Reply:
I removed it to use the one from the project